haproxy.py

```diff
@@ -57,13 +57,25 @@ def watch_request_queues(haproxy_urls, threshold, check_interval):
         # get heartbeat-failure alerts
         pools = fetch_queue_lengths_by_pool(haproxy_urls)
 
+        day_start = datetime.time(8)
```

Comment:
jdubs says it's in PST. Maybe we should start doing UTC going forward as not to worry about daylight savings time and different time zones of people?